addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/RFXComBindingConstants.java

```diff
@@ -72,8 +73,7 @@
     public static final String CHANNEL_RAIN_TOTAL = "rainTotal";
     public static final String CHANNEL_WIND_DIRECTION = "windDirection";
     public static final String CHANNEL_WIND_SPEED = "windSpeed";
-    public static final String CHANNEL_GUST = "gust";
-    public static final String CHANNEL_CHILL_FACTOR = "chillFactor";
```

Comment:
They were unused. Previously the gust value was what was being returned as windSpeed so to avoid breaking existing "working" setups I elected to keep that and add averageWindSpeed.